docs/guides/admin/docs/configuration/basic.md

```diff
@@ -64,7 +64,16 @@ broker is mostly run on the admin server of Opencast but can be run separately.
 For more details about the setup, have a look at the [Apache ActiveMQ configuration guide](message-broker.md).
 
 
-Step 5: Database Configuration
+Step 5: Setting up Elasticsearch
+--------------------------------
+
+Since version 9.0, Opencast requires running an external Elasticsearch instance for indexing purposes.  Typically this
```

Comment:
Can't answer the other discussion for some reason… let's start a new one :+1: 

---

Since when Opencast requires it never matters and mentioning a version is weird as soon as this ends up in the 10.x guide:

```suggestion
Opencast requires Elasticsearch.  Typically this
```

---

I'm also still a bit unsure about the guidance of users. This guide is referenced in all the installation docs, so you should already have gone through them. We are kind of building a circular dependency here. We are also not mentioning Solr here since it just works in the default set-up.

I'm wondering if it wouldn't be better to instruct users in the installation docs to install Elasticsearch on the admin node which makes this step here obsolete. That's what users should do anyway (otherwise, you need a security concept for how to protect Elasticsearch from external access). We then have the additional advanced docs for Elasticsearch for when people want to run it separately like we have for Solr.

I think that would make the installation easier and less complex.

Reply:
Fair enough.  I've removed most of the paragraph since the installation docs all seem to contain instructions to install ES.

After this force push, how do the docs look?